modules/ROOT/pages/introduction/intro-use-exchange.adoc

```diff
@@ -0,0 +1,195 @@
+= Use Anypont Exchange to Discover Connectors, Templates, and Examples
+
```

Comment:
Metadata cannot be in a new line. Remove the extra line.
```suggestion
```
Also, don't specify `:imagesdir: ../../assets/images/`. You don't need it and you might be overwriting the actual imagesdir declaration from the `_attributes.adoc[]` file that you are including.